modules/enable-monitoring-central-helm.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: PROCEDURE
+[id="enable-monitoring-central-helm_{context}"]
+= Monitoring Central using Helm charts
+
+[role="_abstract"]
+You can monitor Central, by changing the Central configuration options in the  the `central-services` Helm chart.
+
+.Procedure
+. Update the `values-public.yaml` configuration file with the following:
++
+[source,yaml]
+----
+exposeMonitoring: true
```

Comment:
I would mention scanner here as well, in analogy to the operator section. To enable monitoring for scanner:
```
scanner.exposeMonitoring: true
```
Also, this Helm flag is not mentioned on https://58057--docspreview.netlify.app/openshift-acs/latest/installing/installing_other/install-central-other.html#central-services-public-configuration-file-scanner_install-central-other.